glide-core/src/client/value_conversion.rs

```diff
@@ -233,6 +233,13 @@ pub(crate) fn expected_type_for_cmd(cmd: &Cmd) -> Option<ExpectedReturnType> {
         b"ZRANK" | b"ZREVRANK" => cmd
             .position(b"WITHSCORE")
             .map(|_| ExpectedReturnType::ZrankReturnType),
+        b"SPOP" => {
```

Comment:
add test

Reply:
add tests to node client

Reply:
@Yury-Fridlyand wdym ?

Reply:
Oh, node has no sets...
Since all node tests pass everything is fine